debian/changelog

```diff
@@ -1,3 +1,16 @@
+pg-auto-failover (1.3.citus-1) stable; urgency=low
+
+  * Fix systemd integration [#212]
```

Comment:
We have empty lines between changelog items.

Reply:
This is automatically filled by `update_os_package.pl`. This is simply the way the changelog is formatted on the pg_auto_failover repo.

Reply:
The debian changeling is supposed to contain entries relevant to the debian packaging, not to the software being packaged. So you could have entries about the improvements you made in the debian packaging, or just one line mentioning it's an official release of the software...

Reply:
See https://www.debian.org/doc/debian-policy/ch-source.html#debian-changelog-debian-changelog for more information and details.

Reply:
I suggest we have the usual single entry here:
```
  * Official release of 1.3.1
```

We will be missing an entry for 1.3 but I guess this is the way it is here, and we are missing several other changelog sections as well.